### PR TITLE
perfschema: Fix nil pointer panic when perfschema is uninitialized

### PR DESCRIPTION
**Pull request by shenli**
In bootstrap stage, ps may be not well initialized. We should make it safe.
@nieyy

### CODE REVIEW

**Comment by nieyy** [comment]
LGTM

**Comment by nieyy** [comment]
Sorry, I missed this case,

File: perfschema/statement_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package perfschema
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/util/testleak"
+	"github.com/pingcap/tidb/util/types"
+)
+
+type testStatementSuit struct {
+	vars map[string]interface{}
```
Comment (zimulala) [inline perfschema/statement_test.go]: `vars` is useful ?

**Comment by zimulala** [comment]
LGTM

**Pull request merged**